test/Garnet.test/Resp/ACL/RespCommandTests.cs

```diff
@@ -2493,18 +2493,68 @@ async Task DoEvalShaAsync(GarnetClient client)
         }
 
         [Test]
-        public async Task ScriptACLsAsync()
+        public async Task ScriptLoadACLsAsync()
         {
             await CheckCommandsAsync(
-                "SCRIPT",
-                [DoScriptAsync],
-                knownCategories: ["slow"]
+                "SCRIPT LOAD",
+                [DoScriptLoadAsync]
             );
 
-            async Task DoScriptAsync(GarnetClient client)
+            async Task DoScriptLoadAsync(GarnetClient client)
             {
                 string res = await client.ExecuteForStringResultAsync("SCRIPT", ["LOAD", "return 'OK'"]);
-                ClassicAssert.AreEqual("57ade87c8731f041ecac85aba56623f8af391fab", (string)res);
+                ClassicAssert.AreEqual("57ADE87C8731F041ECAC85ABA56623F8AF391FAB", (string)res);
```

Comment:
i thought redis returns lowercase sha?

Reply:
Huh, you're right.  I think I must have confused some SE.Redis behavior for plain Redis behavior.